templates/base.html.twig

```diff
@@ -9,8 +9,40 @@
     {% endblock %}
 </head>
 <body>
-
-{% block body %}{% endblock %}
+    <header>
+        <div class="navbar">
+            <div class="actu">
+                Actualités
+            </div>
+            <div class="separator">
+                <img src="{{asset('build/images/vertical-line.png') }}" alt="">
+            </div>
+            <div class="design">
+                Design métallique
+            </div>
+            <div class="logo-navbar">
+                <div class="img-only">
+                    <a href="{{path('home') }}"><img src="{{ asset('build/images/logo-navbar.png') }}" alt="Le logo de la société Amon" width="100%" title="Aller à l'accueil"></a> 
+                </div>
+            </div>
+            <div class="passages">
+                Passages secrets
+            </div>
+            <div class="separator">
+                <img src="{{asset('build/images/vertical-line.png') }}" alt="">
+            </div>
+            <div class="contact">
+                Contact
+            </div>
```

Comment:
And because this is a navigation menu : i.e. a list of links
It should be an html list of links not a div of divs